docker-compose/common-services/anchor-platform-config.yaml

```diff
@@ -0,0 +1,335 @@
+version: 1
+
+host_url: http://host.docker.internal:8080
+
+stellar_network:
+  network: TESTNET
+  network_passphrase: Test SDF Network ; September 2015
+  horizon_url: https://horizon-testnet.stellar.org
+
+callback_api:
+  base_url: http://host.docker.internal:8081
+  auth:
+    type: NONE
+
+platform_api:
+  auth:
+    type: NONE
+  base_url: http://host.docker.internal:8080
+
+languages: en
+
+logging:
+  level: WARN
+  stellar_level: INFO
+
+#data:
+#  type: sqlite
+#  database: anchor-platform
+#  initial_connection_pool_size: 2
+#  max_active_connections: 10
+#  flyway_enabled: true
+#  ddl_auto: update
+
+data:
+  type: postgres
+  server: localhost:5440
+  database: postgres
+#  username: "postgres"
+#  password: "password"
+  initial_connection_pool_size: 3
+  max_active_connections: 10
+  flyway_enabled: true
+  ddl_auto: update
+
+sep1:
+  enabled: true
+  toml:
```

Comment:
The toml can be separated to another file.

Reply:
Done